### DISCUSSION TITLE
GitHub notebook viewer does not show raw cells

### DISCUSSION THREAD

**Discussion by AdeelH** [community/community#41808] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Jupyter notebook cells come in 3 flavors: code, markdown, and _raw_. The GitHub notebook viewer currently does not show raw cells at all. This is very unintuitive behavior.

The notebook viewer is not expected to render them, but they should still be visible as raw text.

Here is an example of a notebook with raw cells (containing reST styling):
- on GitHub (raw cells hidden): https://github.com/azavea/raster-vision/blob/master/docs/usage/tutorials/train.ipynb
- in the web editor (raw cells visible): https://github.dev/azavea/raster-vision/blob/master/docs/usage/tutorials/train.ipynb
- rendered as HTML (raw cells visible): https://docs.rastervision.io/en/latest/usage/tutorials/train.html

### Screenshots
**GitHub**
![image](https://user-images.githubusercontent.com/13014700/207850926-1f459d4c-6be8-46d3-b93c-7da9ca8b6a4a.png)

**Web editor / VS Code**
![image](https://user-images.githubusercontent.com/13014700/207851217-a97ea137-a946-447f-92ef-b9c2913e120e.png)

**HTML**
![image](https://user-images.githubusercontent.com/13014700/207851349-3829143f-d8c5-473e-ba11-b98356a7c5e7.png)

**Comment by jfullstackdev** [comment]
I tried opening them and I cannot compare, can you have a screenshot of that ?

**Reply by AdeelH** [reply]
Just added screenshots. Note the missing headings.

**Reply by zabop** [reply]
I have the same issue, Raw cells still don't show up. Let me know if there is a need for further reproducibility.
